modules/ossm-threescale-applying-external-service-entry-objects.adoc

```diff
@@ -6,21 +6,21 @@
 [id="ossm-threescale-applying-external-service-entry-objects_{context}"]
 = Applying 3scale external ServiceEntry objects
 
-To have the `threescale-wasm-auth` module authorize requests against 3scale, the module must have access to 3scale services. You can accomplish this within {SMProductName} and Istio by applying an external `ServiceEntry` object.
+To have the `threescale-wasm-auth` module authorize requests against 3scale, the module must have access to 3scale services. You can accomplish this within {ProductName} and Istio by applying an external `ServiceEntry` object and a corresponding `DestinationRule` object for TLS configuration to use the HTTPS protocol.
 
-The custom resources set up the service entries for access from within {SMProductShortName} to 3scale Hosted (SaaS) for the backend and system components of the Service Management API and the Account Management API. The Service Management API receives queries for the authorization status of each request. The Account Management API provides API management configuration settings for your services.
+The custom resources (CRs) set up the service entries and destination rules for secure access from within {SMProductShortName} to 3scale Hosted (SaaS) for the backend and system components of the Service Management API and the Account Management API. The Service Management API receives queries for the authorization status of each request. The Account Management API provides API management configuration settings for your services.
 
 .Procedure
 
-* Apply the following external `ServiceEntry` custom resources to your cluster:
+. Apply the following external `ServiceEntry` CR and related `DestinationRule` CR for 3scale Hosted backend to your cluster:
```

Comment:
Both procedure steps are very similar and have only difference is in words "backend" in first step and "system" in second step. I would make these words **bold**  or otherwise distinguish them to stand out a little.

Reply:
Thanks @azgabur That's a good idea based on the previous feedback from Julie.